lib/trento/application/read_models/cluster_read_model.ex

```diff
@@ -17,7 +17,7 @@ defmodule Trento.ClusterReadModel do
   @derive {Jason.Encoder, except: [:__meta__, :__struct__]}
   @primary_key {:id, :binary_id, autogenerate: false}
   schema "clusters" do
-    field :name, :string
+    field :name, :string, default: ""
```

Comment:
If we don't set the default value, when we get a `nil` value in the `name` field (which is the current scenario), it doesn't apply any change in the persistent storage, I guess because ecto works this way.

Reply:
@nelsonkopliku I will add some default value in the `name` field by now. Let's postpone the addition of the ID, as it would requires to change the nice 3x3 grid we have now, and I don't want to play with that by now hehe

Reply:
Here we go (https://github.com/trento-project/web/pull/497/commits/a434de3cb79e5732ef9ab96759983fed31148897):
![image](https://user-images.githubusercontent.com/36370954/167584094-63f064af-0482-4534-b3de-cd941a7bfcde.png)

Reply:
ok makes sense! Thanks!